providers/tentacle.rb

```diff
@@ -168,7 +169,11 @@
     notifies :restart, "windows_service[#{service_name}]", :delayed
   end
 
-  new_resource.updated_by_last_action(actions_updated?([register_instance]))
+  service = windows_service service_name do
```

Comment:
Not sure I follow, are you saying change this:

   new_resource.updated_by_last_action(actions_updated?([register_instance, service]))

to this? 

    new_resource.updated_by_last_action(actions_updated?([register_instance]))